pkg/annotations/annotations.go

```diff
@@ -8,6 +8,9 @@ const (
 	// UsernsMode is the user namespace mode to use
 	UsernsModeAnnotation = "io.kubernetes.cri-o.userns-mode"
 
+	// CgroupRW specifies mounting v2 cgroups as an rw filesystem.
+	CgroupRWAnnotation = "io.kubernetes.cri-o.cgroup-rw"
```

Comment:
I feel like the name implies this works for all cgroups. do we want it to be Cgroup2RW?